sql/pgtap.sql.in

```diff
@@ -2376,6 +2376,17 @@ RETURNS TEXT AS $$
     SELECT col_has_check( $1, $2, 'Column ' || quote_ident($1) || '(' || quote_ident($2) || ') should have a check constraint' );
 $$ LANGUAGE sql;
 
+-- table_constraint_checksum ( schema, table, conname, contype, checksum, description)
+CREATE OR REPLACE FUNCTION table_constraint_checksum (NAME, NAME, NAME, char, text, text)
+RETURNS TEXT AS $$
+  SELECT is(md5 ( pg_get_constraintdef(oid)), $5, $6)
+  FROM pg_constraint
+  WHERE connamespace=(SELECT oid FROM pg_namespace WHERE nspname=$1)
```

Comment:
Maybe just use `$1::regnamespace`. Or for that matter, maybe the function should just accept `regnamespace`. At some point we should really have function versions that accept `reg*` pseudo-types instead of forcing people to segregate schema from object name.